travel/travel.py

```diff
@@ -25,6 +25,17 @@
 from .res_config import get_basic_passenger_limit
 
 
+def _get_travel_states(self, cr, uid, ids=None, context=None):
```

Comment:
@pedrobaeza This is related to a real bug, and there is an issue. The issue is internal to Savoir-faire Linux, not public in the web. I don't see why a comment/reference is required here. If you write a function field of type selection, you must supply a list of selections. This is standard Odoo/Openerp behavior.